reporter/src/main/kotlin/gitlab/LicenseModelMapper.kt

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.reporter.gitlab
```

Comment:
This should probably go to the existing "model" sub-package, like the other model mappers.

Reply:
The "model" sub-package currently contains only classes for the "evaluated model" and I believe it makes sense to keep a separate package for it.

Reply:
Ok, the "model" sub-package might not be the right place, but probably the "utils" package then is, as that's also where classes like `AmazonOssAttributionBuilderService` and `SpdxDocumentModelMapper` are located. At least in the current structure.

Maybe it's a good idea to switch to per-reporter sub-packages, but then we should clean up the existing code base in that regard first, and move e.g. the above-mentioned classes to "amazon" and "spdx" sub-packages first.

Reply:
Ok, I'll address it in a separate commit on top.

Reply:
I like the actual grouping much more because of cohesion, anyhow you're right that it'd be inconsistent.

Reply:
If you refer to moving classes in the existing code base, please do that in a separate commit "at bottom", i.e. make that the first commit, then add your new files.

Reply:
> I like the actual grouping much more because of cohesion, anyhow you're right that it'd be inconsistent.

Right, I don't really care so much about for which grouping we go, but it should be consistent.

Reply:
Actually looking at it again I find that the "model" package contains

- the model classes for the evaluated model
- the mapper for the evaluated model

This means that it is already inconsistent. Thus I believe it should be fine to keep my PR as is as it was fine for the evaluated model as well. I'll file a follow up PR then to move the other things to separate packages.
Do you agree with this argumentation?

Reply:
I'm not 100% following your argumentation why it's inconsistent to put evaluated *model* classes into a *model* package. And I also prefer to first make things consistent and then add new stuff, instead of temporarily making inconsistency worse by first adding new stuff and then cleaning up. But I'm basically fine if in the end all will be consistent.

Reply:
Well, you were arguing that my implementation was inconsistent with existing stuff.

I was arguing that my implementation is consistent with the evaluated model implementation, but not with other reports - which implies that existing stuff is inconsistent. Anyhow, I'm not sure if it's clearer now but it doesn't matter anymore as we found an agreement.